docs/resources/synthetic_monitoring_check.md

```diff
@@ -559,6 +561,96 @@ Optional:
 
 
 
+<a id="nestedblock--settings--multihttp"></a>
+### Nested Schema for `settings.multihttp`
+
+Optional:
+
+- `entries` (Block List) (see [below for nested schema](#nestedblock--settings--multihttp--entries))
+
+<a id="nestedblock--settings--multihttp--entries"></a>
+### Nested Schema for `settings.multihttp.entries`
+
+Optional:
+
+- `checks` (Block List) Assertions to make on the request response (see [below for nested schema](#nestedblock--settings--multihttp--entries--checks))
+- `request` (Block Set, Max: 1) An individual MultiHTTP request (see [below for nested schema](#nestedblock--settings--multihttp--entries--request))
+- `variables` (Block List) Variables to extract from the request response (see [below for nested schema](#nestedblock--settings--multihttp--entries--variables))
+
+<a id="nestedblock--settings--multihttp--entries--checks"></a>
+### Nested Schema for `settings.multihttp.entries.checks`
+
+Required:
+
+- `type` (Number) The type of assertion to make. TEXT = 0, JSON_PATH_VALUE = 1, JSON_PATH_ASSERTION = 2, REGEX_ASSERTION = 3
+
+Optional:
+
+- `condition` (Number) The condition of the assertion. NOT_CONTAINS = 1, EQUALS = 2, STARTS_WITH = 3, ENDS_WITH = 4, TYPE_OF = 5, CONTAINS = 6
+- `expression` (String) The expression of the assertion. Should start with $.
```

Comment:
Does the expression actually have to start with $? I'm not sure it does

Reply:
I'm not sure why it says it should start with `$`...

Reply:
Ah, I see what you mean now.

For JSON_PATH_ASSERTION and JSON_PATH_VALUE, the JSONPath starts with `$.` because that's the root of the JSON object.

Reply:
The thing is, it works without $., and I think maybe it won't work with $.